gno.land/cmd/gnoland/testdata/versioning.txtar

```diff
@@ -0,0 +1,81 @@
+# test package versioning
+
+## start a new node
+gnoland start
+
+## add `hello`(v1) package located in $WORK/helloworld/v1
+gnokey maketx addpkg -pkgdir $WORK/helloworld/v1 -pkgpath gno.land/p/helloworld/hello -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+## add `hello`(v2) package located in $WORK/helloworld/v2
+gnokey maketx addpkg -pkgdir $WORK/helloworld/v2 -pkgpath gno.land/p/helloworld/hello -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+
+## add `foo` realm; imports pkg `hello` v1
+gnokey maketx addpkg -pkgdir $WORK/foo -pkgpath gno.land/r/temp/foo -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+
+## add `bar` realm; imports pkg `hello` v2
+gnokey maketx addpkg -pkgdir $WORK/bar -pkgpath gno.land/r/temp/bar -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+
+## execute Render from `foo` realm
+gnokey maketx call -pkgpath gno.land/r/temp/foo -pkgversion v0.0.1 -func Render -gas-fee 1000000ugnot -gas-wanted 2000000 -args '' -broadcast -chainid=tendermint_test test1
```

Comment:
```suggestion
gnokey maketx call -pkgpath gno.land/r/temp/foo@v0.0.1 -func Render -gas-fee 1000000ugnot -gas-wanted 2000000 -args '' -broadcast -chainid=tendermint_test test1
```
If we use explicit versioning, I suggest something like this.

Reply:
```suggestion
gnokey maketx call -pkgpath gno.land/r/temp/foo@v1 -func Render -gas-fee 1000000ugnot -gas-wanted 2000000 -args '' -broadcast -chainid=tendermint_test test1
```

Or, even better, with a shorter version.

Reply:
Implemented the first suggestion. As it keeps the versioning similar to Golang/go.mod.